.github/CODEOWNERS

```diff
@@ -5,6 +5,7 @@
 /utilities/ @Azure/avm-core-team-technical-bicep
 /avm/ptn/aca-lza/hosting-environment/ @Azure/avm-ptn-acalza-hostingenvironment-module-owners-bicep @Azure/avm-module-reviewers-bicep
 /avm/ptn/ai-platform/baseline/ @Azure/avm-ptn-aiplatform-baseline-module-owners-bicep @Azure/avm-module-reviewers-bicep
+/avm/ptn/app-service-lza/hosting-environment/ @Azure/avm-ptn-appservicelza-hostingenvironment-module-owners-bicep @Azure/avm-module-reviewers-bicep
```

Comment:
Currently shows errors. Reasons could be
- team does not exist
- team is not in the correct hierachy
- team's place in the hierachy was not yet confirmed by the parent owner team.
[ref](https://azure.github.io/Azure-Verified-Modules/specs/bcp/res/#:~:text=Grant%20Permissions%20%2D%20Bicep-,Team%20memberships,-Note)

Reply:
Team did not exist yet. I've created the team created in the Azure org

Reply:
Much appreciated :) Please also ensure that it is added to the correct parent team to enable you to approve PRs for your module down the line.
For the owner team: `avm-technical-reviewers-bicep`
For the contributor team: `avm-module-contributors-bicep`